### PR TITLE
Introduce Zeit (now) auto deploy

### PR DESCRIPTION
**Pull request by gyndav**
## Business rationale
In order to ease reviewing when adding new components to Storybook, let's use **Zeit** in order to **automatically deploy each PR** on an isolated environment.

## Changes

* introduce Zeit `now`

### CODE REVIEW

File: docs/how-to-review-new-components.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+## How-to review new components
+
+Each pull request will be automatically deploy on [Zeit](https://zeit.co) via `now`,
+producting a unique URL such as https://flamingo-git-gyndav-featzeit.heetch-eng.now.sh
```
Comment (ugogo) [inline docs/how-to-review-new-components.md]: How is this `flamingo-git-gyndav-featzeit` built?
Is this related to the branch's name?

File: now.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "flamingo",
+    "version": 2,
```
Comment (ugogo) [inline now.json]: That's the `now` version, not Flamingo's one right?

File: now.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    "builds": [
+      {
+        "src": "build.sh",
```
Comment (ugogo) [inline now.json]: Should we wrap this into a `script/` folder?
We might have more of this latter, wdyt?

File: now.json (hunk 2 above)
Comment (gyndav) [reply now.json]: Yup https://zeit.co/docs/v2/advanced/configuration#version

File: docs/how-to-review-new-components.md (hunk 1 above)
Comment (gyndav) [reply docs/how-to-review-new-components.md]: What do you mean? I'm afraid I don't really understand the underlying question there.
If you meant the name, indeed, sounds like `project-name-platform-branch_name`. Yet another alias, that we can prevent. The original one here was https://flamingo-fw5vr9suf.now.sh/

File: now.json (hunk 3 above)
Comment (gyndav) [reply now.json]: Agreed. Done. ✅

File: docs/how-to-review-new-components.md (hunk 1 above)
Comment (ugogo) [reply docs/how-to-review-new-components.md]: Yeah sorry I was talking the naming of this URL
`project-name-platform-branch_name` sounds good 👍

**Comment by gyndav** [comment]
@ugogo I'm not really fond of what the config and build files became with introducing `scripts`dir ...

Platform assumes only **relative path** (not absolute one).
Do you mind if we rollback the `scripts` directory change?

**Comment by ugogo** [comment]
@gyndav Ok if we can rename the `.sh` script to `now.build.sh` (or something like that) 🙂

**Comment by gyndav** [comment]
@ugogo done with `now-build.sh` (sorry that's the Shell default separator 😅 )

**Comment by ugogo** [comment]
Not the best but we can live with that 🙂 
Thanks for the help @gyndav!

**Comment by pppemo** [comment]
Cool is this good to be merged now @gyndav ?

**Comment by gyndav** [comment]
for sure @pppemo 🔥
